package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hemilabs/token-list",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
Shouldn't this be 1.0.2 instead?

Reply:
I was on the fence wheter to make a minor or a patch update. I don't think it is specifically a bug. I am open to any option

Reply:
I could consider it a bug as the name was wrong and now is fixed.

Reply:
@gabmontes Updated - I also replaced the tag